src/phylo/distances.jl

```diff
@@ -170,7 +232,58 @@ end
 
 # K80 Distance computation.
 
+<<<<<<< HEAD
```

Comment:
Sounds awesome! Excuse the uninvited interjections, for some reason the diffs display in email alerts and though I'd let you know in case you missed it! Sorry for the noise.